src/components/com_tjdashboard/administrator/models/dashboard.php

```diff
@@ -97,14 +97,35 @@ public function save($data)
 			return false;
 		}
 
+		$result = $dashboard->save();
+
 		// Store the data.
-		if (!$dashboard->save())
+		if (!$result)
 		{
 			$this->setError($dashboard->getError());
 
 			return false;
 		}
 
-		return $dashboard->dashboard_id;
+		$this->setState('dashboard.dashboard_id', $result);
```

Comment:
@monikaatvowel you don't require both the statement (If you are using state then don't return the value)

Reply:
now just using state